packages/blocks/button/package.json

```diff
@@ -41,7 +41,7 @@
     "cross-env": "^7.0.3",
     "css-loader": "^6.7.0",
     "html-webpack-plugin": "^5.5.0",
-    "mock-block-dock": "0.0.5",
+    "mock-block-dock": "0.0.6",
```

Comment:
Upgrading `mock-block-dock` and pinning `blockprotocol` was not necessary, but I did not just in case given the opportunity.